rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -37,6 +40,28 @@ def preprocess_pdf_content(content: str):
     return content
 
 
+def _load_pdf_using_pypdf(file_path: str) -> List[Document]:
+    loader = PyPDFLoader(file_path=file_path)
+    return loader.load()
+
+
+def _load_pdf_using_unstructured(file_path: str) -> List[Document]:
+    elements = partition_pdf(filename=file_path, strategy="ocr_only")
+    element_meta = elements[0].to_dict()["metadata"] if elements else {}
```

Comment:
Severity: Major - Consider adding error handling for cases where 'elements' might be empty. This will prevent potential runtime errors when accessing 'elements[0]'.